src/templates/color-picker.hbs

```diff
@@ -27,11 +27,17 @@
     <form {{bind-attr class=":color-picker-custom-form isCustomColorValid:valid:invalid"}}>
       <div class="form-group form-group-sm">
         <div class="input-group">
-          <span {{bind-attr class=":input-group-addon
-            :color-picker-custom-preview isCustomColor:active"
-            style="customColorCSS"}}></span>
+          {{#if isCustomColorValid}}
+            {{view Ember.Widgets.ColorPickerCell
+              colorBinding="customColor"
+            }}
+          {{else}}
```

Comment:
There's no validation check in `customColorCSS` so it will be `background-color: #{input}` regardless of what the input is.